engine/execution/rpc/engine.go

```diff
@@ -256,7 +256,8 @@ func (h *handler) GetEventsForBlockIDs(_ context.Context,
 	}
 
 	return &execution.GetEventsForBlockIDsResponse{
-		Results: results,
+		Results:              results,
```

Comment:
Here and below: Does this mean that events will be returned as CCF from APIs?

Reply:
> Does this mean that events will be returned as CCF from APIs?

Yes, I think it means `result` contains events encoded in CCF.  AN will convert CCF encoded events if `EventEncodingVersion` is `execution.EventEncodingVersion_CCF_V0`.

@koko1123 This is related to the endpoint fix you contributed, can you confirm/correct our understanding?  Thanks!

Reply:
I see, nice! I just wanted to make sure nothing changes for clients, i.e. these are backwards-compatible changes 👍

Reply:
correct! otherwise the protobuf defaults to `EventEncodingVersion_JSON_CDC_V0`